### PR TITLE
Migrate project#new_package

### PR DESCRIPTION
**Pull request by dmarcoux**
Preview:
create_package_form

The modal which is now using the same partial as `project#new_package` didn't change, except for marking the `name` field as required.

create_package_modal

/rant

We have 2 ways to create packages. One with a modal and with project#new_package when a validation fails in the modal. Another case of a modal which is useless.

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Migrate Package#new_package view](https://trello.com/c/lD7JeW9v/1272-migrate-packagenewpackage-view)

**Comment by dmarcoux** [comment]
Review app is broken because I put a hash (`#`) in my branch name.

Well, it's sad, but test it locally. Nothing needs to be setup, so it's really easy. Go to http://localhost:3000/project/new_package/home:Admin

File: src/api/app/views/webui2/webui/project/new_package.html.haml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- @pagetitle = "Create package for #{@project}"
+
+.card.mb-3
+  = render partial: 'tabs', locals: { project: @project }
+  .card-body
+    %h3= @pagetitle
+    = render partial: 'create_package_form', locals: { project: @project }
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/project/new_package.html.haml]: The cancel button doesn't have any sense on this page. Could you remove it?, please :smile_cat:

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/project/new_package.html.haml]: True! Please have another look. I updated the screenshots in the PR's description.

**Pull request merged**